src/components/forms/event-type-form.js

```diff
@@ -373,6 +373,31 @@ class EventTypeForm extends React.Component {
                             </div>
                         </div>
                     </div>
+                    <hr/>
+                    <div className="row form-group">
+                        <div className="col-md-4">
+                            <label> {T.translate("edit_event_type.min_duration")}</label>
+                            <Input
+                                id="min_duration"
+                                type="number"
+                                value={entity.min_duration}
+                                onChange={this.handleChange}
+                                className="form-control"
+                                error={hasErrors('min_duration', errors)}
+                            />
+                        </div>
+                        <div className="col-md-4">
+                            <label> {T.translate("edit_event_type.max_duration")}</label>
```

Comment:
lets add an info button with a tooltip ( title ) that states ( its only applies to track chairs )

Reply:
done